openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -0,0 +1,190 @@
+"""Utilities for working with ID tokens."""
+import json
+from time import time
+
+from django.conf import settings
+from jwkest import jwk
+from jwkest.jws import JWS
+
+from edx_django_utils.monitoring import set_custom_metric
+from openedx.core.djangoapps.oauth_dispatch.toggles import ENFORCE_JWT_SCOPES
+from student.models import UserProfile, anonymous_id_for_user
+
+
+def create_api_client_jwt(user, secret=None, aud=None, additional_claims=None):
+    """
+    To create JWTs for API clients that are making REST API calls to other edX services.
+
+    Arguments:
+        user (User): User for which to generate the JWT.
+
+    Deprecated Arguments (to be removed):
```

Comment:
Would be great to have a JIRA ticket ID (or some other link) that provides more details about why these can go away and what are the blockers to removing them.

Reply:
skip.  This is all part of the longer effort to cleanup authn.